felix/bpf-gpl/parsing.h

```diff
@@ -10,6 +10,66 @@
 #define PARSING_ALLOW_WITHOUT_ENFORCING_POLICY 2
 #define PARSING_ERROR -1
 
+#define MAX_EXTENSIONS 2
+
+static CALI_BPF_INLINE int parse_packet_ipv6(struct cali_tc_ctx *ctx, int iterations) {
+	__u8 next_header = ipv6_hdr(ctx)->nexthdr;
+
+	for (int i = 0; i < iterations; i++) {
```

Comment:
do you know the value upfront or is it just a static max decided at compile time? Does it need to be provided from outside?

Reply:
Move this part to another PR.